python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -69,7 +69,7 @@ def partition_for_arm_compute_lib(mod, params=None, **opts):
         ]
     )
 
-    return seq(mod)
+    return seq(mod), None
```

Comment:
With the new return type, please update the docstrings.

This applies to all `partition_for_*` functions.